packages/docs/src/utilities/utils.ts

```diff
@@ -1,25 +1,31 @@
 // File for documentation helper functions
 
+function getUIElementNameFromPath (path: string): string {
+  // 'src\page-configs\ui-elements\component-name' --> 'component-name'
```

Comment:
Can you make it explicit that's a windows issue?
Pretty important in my opinion.